README.md

```diff
@@ -111,6 +111,53 @@ $ curl -X POST -d '{"old_password":"dmytro.striletskyi.1337", "new_password":"dm
 | old_password | Input arguments validation | This field is required.                            | 400         |
 | new_password | Input arguments validation | This field is required.                            | 400         |
 
+* `POST | /user/password/recovery` - password recovery for an existing user by his email.
```

Comment:
`Request password recovery for an existing user by his e-mail address. Send the recovery link to the e-mail address.`

Reply:
@dmytrostriletskyi, done.